Installation/include/CGAL/config.h

```diff
@@ -152,6 +152,10 @@
     defined(BOOST_NO_CXX11_HDR_ARRAY) || BOOST_VERSION < 104000
 #define CGAL_CFG_NO_CPP0X_ARRAY 1
 #endif
+#if defined(BOOST_NO_CXX11_HDR_UNORDERED_SET) || \
+    defined(BOOST_NO_CXX11_HDR_UNORDERED_MAP)
+#define CGAL_CFG_NO_CPP0X_UNORDERED 1
+#endif
```

Comment:
[Sébastien is right][*]. You should add 
```C++
                               || \
BOOST_NO_0X_HDR_UNORDERED_MAP  || \
BOOST_NO_0X_HDR_UNORDERED_SET
```

  [*]: http://www.boost.org/doc/libs/1_48_0/libs/config/doc/html/boost_config/boost_macro_reference.html#boost_config.boost_macro_reference.macros_that_describe_possible_c__0x_features